Update Get-CsOnlineUser.md (#9357)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

- UserDirSyncEnabled
- TeamsWorkLoadPolicy
- VoiceRoutingPolicy

> - LastName (available in Teams PowerShell Module 4.2.0 and later)

Reviewer: @isrumnon  - is this not there in 4.2.1? if yes, let's add that instead of full removal

In the Teams PowerShell Module version 3.0.0 or later, the format of the AssignedPlan and ProvisionedPlan attributes has changed from XML to JSON array. Previous XML filters (For example, `-Filter "AssignedPlan -eq '<some-xml-string>'"`) will no longer work. Instead, you need to update your filters to use one of the following formats:

> - All users with an AssignedPlan matches MCO: `-Filter "AssignedPlan -eq 'MCO'"`

Reviewer 2: ```suggestion
- All users with an AssignedPlan that matches MCO: `-Filter "AssignedPlan -eq 'MCO'"`
```

## INPUTS

## NOTES

> Note: The the mentioned changes in the above sections including changes under "-Identity" and "-Filter" paramters are not applicable for customers and tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. 

Reviewer 3: ```suggestion
The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
```
